tests/filescan.test

```diff
@@ -31,7 +31,7 @@ proc IncrName {Name args} {
     set Begin [csubstr $Name 0 $Last]
     set Digit [cindex $Name $Last]
     set Recurse 0
-    case $Digit in {
+    switch $Digit in {
```

Comment:
Should the unnecessary `in` be removed entirely?

Reply:
The case command was deprecated in previous versions and it has been removed entirely in Tcl9.0.